sdk/src/wallet/error.rs

```diff
@@ -42,6 +42,9 @@ pub enum Error {
     /// Insufficient funds to send transaction.
     #[error("address owns insufficient funds: {required} base unit required, but {available} base unit available")]
     InsufficientFunds { available: u64, required: u64 },
+    /// Insufficient funds to send transaction.
+    #[error("account has insufficient block issuance credit: {required} required, but {available} available")]
+    InsufficientBIC { available: i128, required: u64 },
```

Comment:
I think we usually don't put acronyms in uppercase in variants

Reply:
\>:(